zubhub_backend/zubhub/creators/models.py

```diff
@@ -82,9 +82,22 @@ def save(self, *args, **kwargs):
 
 
 class Setting(models.Model):
+    WHATSAPP = 1
+    EMAIL = 2
+    SMS = 3
+
+    CONTACT_CHOICES = (
+        (WHATSAPP, 'WHATSAPP'),
+        (EMAIL, 'EMAIL'),
+        (SMS, 'SMS')
+    )
+
     creator = models.OneToOneField(
         Creator, on_delete=models.CASCADE, primary_key=True)
     subscribe = models.BooleanField(blank=True, default=False)
+    contact = models.PositiveSmallIntegerField(
```

Comment:
In PR #306, @zorazrr is working on creating a helper function that will call one of the task functions (send_text, send_whatsapp, send_email) based on a user's channel preferences so that other functions just have to call this helper function and not worry about that detail. This function will also add the notification to the notifications model appropriately. My plan was to add this function in a utils.py file in the notifications folder. Do you feel that would be the right place for it?